src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -501,6 +447,11 @@ def convert_from_pytorch(pt_state: Dict, config: BertConfig) -> Dict:
                 key = key.replace("weight", "kernel")
                 jax_state[key] = tensor
 
+            if "decoder.weight" in key:
```

Comment:
To make `BartForMaskedLM` pass the integration tests.

Reply:
We will need something more general in the future but this can do for now.